Find a functionally equivalent code:android.media.SubtitleTrack.hide:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -5,10 +5,13 @@
     if (mTimeProvider != null) {
         mTimeProvider.cancelNotifications(this);
     }
-    getRenderingWidget().setVisible(false);
+    RenderingWidget renderingWidget = getRenderingWidget();
+    if (renderingWidget != null) {
+        renderingWidget.setVisible(false);
+    }
     mVisible = false;
 }

